cmd/virt-manifest/virt-manifest.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/emicklei/go-restful"
+	"github.com/libvirt/libvirt-go"
+
+	"kubevirt.io/kubevirt/pkg/logging"
+	"kubevirt.io/kubevirt/pkg/virt-handler/virtwrap"
+	"kubevirt.io/kubevirt/pkg/virt-manifest/rest"
+)
+
+func main() {
+	logging.InitializeLogging("virt-manifest")
+	libvirt.EventRegisterDefaultImpl()
+	libvirtUri := flag.String("libvirt-uri", "qemu:///system", "Libvirt connection string.")
+	libvirtUser := flag.String("user", "", "Libvirt user")
+	libvirtPasswd := flag.String("pass", "", "Libvirt password")
+	listen := flag.String("listen", "0.0.0.0", "Address where to listen on")
+	port := flag.Int("port", 8186, "Port to listen on")
+	flag.Parse()
+
+	log := logging.DefaultLogger()
+	log.Info().Msg("Starting virt-manifest server")
+
+	log.Info().Msg("Connecting to libvirt")
+	// FIXME: this connection should be closed immediately
+	// but there's currently an error preventing that
+	domainConn, err := virtwrap.NewConnection(*libvirtUri, *libvirtUser, *libvirtPasswd, 60*time.Second)
+	if err != nil {
+		log.Error().Reason(err).Msg("cannot connect to libvirt")
+		panic(fmt.Sprintf("failed to connect to libvirt: %v", err))
```

Comment:
We should think about a Fatalf equivalent, which first logs and then `panics`:

https://github.com/golang/glog/blob/master/glog.go#L1147